src/olympia/blocklist/tests/test_cron.py

```diff
@@ -259,15 +487,34 @@ def test_upload_stash_unless_missing_base_filter(self):
             )
             in self.mocks['olympia.blocklist.cron.upload_filter.delay'].call_args_list
         )
-        assert mlbf.storage.exists(mlbf.filter_path())
-
-    @mock.patch('olympia.blocklist.cron.BASE_REPLACE_THRESHOLD', 1)
+        assert mlbf.storage.exists(mlbf.filter_path(BlockType.BLOCKED))
+
+        with override_switch('enable-soft-blocking', active=True):
+            upload_mlbf_to_remote_settings()
+            assert not mlbf.storage.exists(mlbf.filter_path(BlockType.SOFT_BLOCKED))
+            assert (
+                mock.call(
+                    self.current_time,
+                    # Even if updating a soft block and the switch is active
+                    # don't expect a soft block filter update until we
+                    # implement support for that
+                    filter_list=[BlockType.BLOCKED.name],
+                    create_stash=False,
+                )
+            ) in self.mocks['olympia.blocklist.cron.upload_filter.delay'].call_args_list
+
+    # TODO: add test for soft blocks and ensure stash/filter compatibility
```

Comment:
Is this TODO addressed in the next PR?

Reply:
Covered by https://github.com/mozilla/addons/issues/15200